backend/core/.env.template

```diff
@@ -18,3 +18,4 @@ PARTNERS_PORTAL_URL=http://localhost:3001
 CORS_ORIGINS=["http://localhost:3000", "http://localhost:3001"]
 CORS_REGEX=["test1", "test2"]
 AFS_PROCESSING_CRON_STRING=0 * * * *
+SHOW_MANDATED_ACCOUNTS=FALSE
```

Comment:
@emilyjablonski @YazeedLoonat @ludtkemorgan Seems like this is the first feature toggle / boolean env variable in the template. How do y'all prefer to handle checking if a env var is true or false? I just did a simple string compare, but happy to change it if you have a preference!

Reply:
Sounds good, just pushed up that change!